### ISSUE TITLE
Hack Week: June 8, 2020 master issue

### ISSUE THREAD

**Issue by mindgraffiti** [wordpress-mobile/WordPress-iOS#14264]
- [x] Quick Start v2: VoiceOver: references to "Tap" in toast messages should be read as "Double tap" #12129. PR https://github.com/wordpress-mobile/WordPress-iOS/pull/14263

- [x] Quick Start v2: VoiceOver: the completed cells state is not communicated #12128. PR https://github.com/wordpress-mobile/WordPress-iOS/pull/14265 

- [x] Quick Start: Dismiss Notices using clearWithTag #11394. PR https://github.com/wordpress-mobile/WordPress-iOS/pull/14266

- [x] Quick Start v2: "Follow other sites" misplaced indicator #14268. PR https://github.com/wordpress-mobile/WordPress-iOS/pull/14276

- [x] Quick Start: VoiceOver support missing from Next Steps #12964. PR #14289

- [x] Quick Start v2: VoiceOver: consider providing a hint on what will happens after activating the tour #12130. Analysis made, no PR.
